src/main/java/com/gooddata/sdk/common/collections/PageRequest.java

```diff
@@ -41,9 +42,9 @@
      * <p>
      * Use {@link #getPageUri(UriComponentsBuilder)} if you want to get concrete page URI and don't have URI template.
      *
-     * @param uriBuilder URI builder used for constructing page URI
+     * @param replacer URI builder used for constructing page URI
```

Comment:
"Replacer" sounds to me like something that is used to replace something else. But this argument contains uri that will be replaced (or its parts will be replaced). I would name it e.g. `uriToBeReplaced` ;)